floyd/client/data.py

```diff
@@ -72,10 +74,19 @@ def create(self, data):
             return response.json().get("id")
 
     def get(self, id):
-        response = self.request("GET",
-                                "{}{}".format(self.url, id))
-        data_dict = response.json()
-        return Data.from_dict(data_dict)
+        try:
+            response = self.request("GET",
+                                    "{}{}".format(self.url, id))
+            data_dict = response.json()
+            return Data.from_dict(data_dict)
+        except ClickException as e:
+            floyd_logger.info("Data {}: {}".format(id, e.message))
+            return None
+        except HTTPError as e:
```

Comment:
This class extends the `FloydHttpClient` which takes care of exception handling.